packages/plugins/typescript/msw/tests/msw.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { parse } from 'graphql';
+import { plugin } from '../src';
+
+describe('msw', () => {
+  const documents = [{ document: parse(`query User { name }`) }, { document: parse(`mutation UpdateUser { name }`) }];
+
+  it('Should generate mocks based on queries and mutations', async () => {
+    const result = await plugin(null, documents, {});
+    expect(result.content).toContain(
+      `export const userQueryHandler = (resolver: ResponseResolver<GraphQLRequest<UserQueryVariables>, GraphQLContext<UserQuery>, any>) =>
+  graphql.query<UserQuery, UserQueryVariables>(
+    'User',
+    resolver
+  )`
+    );
+    expect(result.content).toContain(
+      `export const updateUserMutationHandler = (resolver: ResponseResolver<GraphQLRequest<UpdateUserMutationVariables>, GraphQLContext<UpdateUserMutation>, any>) =>
+  graphql.mutation<UpdateUserMutation, UpdateUserMutationVariables>(
+    'UpdateUser',
+    resolver
+  )`
+    );
+    expect(result.prepend).toContain(`import { graphql, ResponseResolver, GraphQLRequest, GraphQLContext } from 'msw'`);
+  });
+
+  it.only('Should generate a link with an endpoint', async () => {
+    const link = { name: 'api', endpoint: 'http://localhost:3000/graphql' };
+    const result = await plugin(null, documents, {
+      link,
+    });
+    expect(result.content).toContain(`const api = graphql.link('http://localhost:3000/graphql')`);
+
+    expect(result.content).toContain(
+      `export const updateUserMutationApiHandler = (resolver: ResponseResolver<GraphQLRequest<UpdateUserMutationVariables>, GraphQLContext<UpdateUserMutation>, any>) =>
+  api.mutation<UpdateUserMutation, UpdateUserMutationVariables>(
+    'UpdateUser',
+    resolver
+  )`
+    );
+  });
+});
```

Comment:
Can you please extend this test suite to cover some more cases? Two test cases seem few for a new plugin. Also there is a `it.only` on one of the tests

Reply:
The `it.only` was removed 👍🏼 . Perhaps, adding [`no-only-tests` ESLint rule](https://www.npmjs.com/package/eslint-plugin-no-only-tests) would help.

I added a few assertions and one test for the generated JSDoc. Let me know if you have something else to test in mind. The plugin is very simple.

Reply:
@smhmd Feel free to send a PR that sets this up 💯